core/src/transformer.ts

```diff
@@ -36,6 +43,21 @@ function Transformer<Model, TransformedModel>(
           };
         }
       });
+    } else if (fieldsToBuild !== false) {
```

Comment:
We now build all buildable data model properties by default so we don't need to specify an array of those ones at the data model level.

Reply:
Nit: Would it make sense to "invert" the logic? If some fields are NOT supposed to be built, we would need to explicitly define them.

If so, we can replace the property `fieldsToBuild` with something like `omitFieldsToBuild`.